# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/CSS2/lists/list-style-image-004.xht.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/CSS2/lists/list-style-image-004.xht.ini
--- a/testing/web-platform/meta/css/CSS2/lists/list-style-image-004.xht.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/CSS2/lists/list-style-image-004.xht.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,7 +1,7 @@
 [list-style-image-004.xht]
   expected:
     if not debug and (os == "win") and not webrender and (processor == "x86_64"): ["PASS", "FAIL"]
+    if not debug and (os == "android") and not webrender: ["PASS", "FAIL"]
     if not debug and (os == "linux") and not fission: ["PASS", "FAIL"]
-    if not debug and (os == "linux") and fission: ["FAIL", "PASS"]
-    if not debug and (os == "android") and not webrender: ["PASS", "FAIL"]
+    if not debug and (os == "linux") and fission: ["PASS", "FAIL"]
     if not debug and (os == "mac"): ["PASS", "FAIL"]